src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -95,4 +125,12 @@ public Optional<EstimatedRobotPose> getEstimatedGlobalPose(Pose2d prevEstimatedR
         m_photonPoseEstimator.setReferencePose(prevEstimatedRobotPose);
         return m_photonPoseEstimator.update();
     }
+
+	public Optional<Pose3d> getTagPose(int tagID){
```

Comment:
Hmm, I am not sure I want this public. At the moment, I don't know why any routine outside of Vision should know anything about the AprilTag number. They should be asking a "question" and getting back the Pose (for instance).

Reply:
And most routines would want Pose2d anyway.